pkg/sql/parser/builtins.go

```diff
@@ -246,6 +247,40 @@ var Builtins = map[string][]Builtin{
 		},
 	},
 
+	"from_ip": {
+		Builtin{
+			Types:      ArgTypes{TypeBytes},
+			ReturnType: TypeString,
+			fn: func(_ *EvalContext, args DTuple) (Datum, error) {
+				ipstr := args[0].(*DBytes)
+				nboip := net.IP(*ipstr)
+				sv := nboip.String()
+				// if nboip has a length of 0, sv will be "<nil>"
+				if sv == "<nil>" {
```

Comment:
One last question, can you briefly explain here in the PR what is the rationale for returning NULL on error, instead of a conversion error?